integration/cli/build.configequality.test.ts

```diff
@@ -0,0 +1,424 @@
+      beforeAll(async () => {
+        // set configuration in package.json
+        const config = await resource.fromFile(packageJson);
+        config.description = "testdesc";
+        config.pepr = {
+          ...config.pepr,
+          peprVersion: "1.2.3",
+          webhookTimeout: 11,
+          onError: "reject",
+          alwaysIgnore: {
+            namespaces: ["garbage"],
+          },
+          logLevel: "warning",
+          env: {
+            TEST: "env",
+          },
+          customLabels: {
+            namespace: {
+              test: "test",
+              value: "value",
+            },
+          },
+          rbacMode: "scoped",
+          rbac: [
+            {
+              apiGroups: ["test"],
+              resources: ["tests"],
+              verbs: ["get", "list"],
+            },
+          ],
+        };
+        await fs.writeFile(packageJson, JSON.stringify(config, null, 2));
+
+        // translate package.json values into ModuleConfig
+        peprUuid = `pepr-${config.pepr.uuid}`;
+        moduleConfig = config.pepr;
+        moduleConfig.appVersion = config.version;
+        moduleConfig.description = config.description;
+
+        // build module
+        const build = await pepr.cli(testModule, { cmd: `pepr build` });
+        expect(build.exitcode).toBe(0);
+        expect(build.stderr.join("").trim()).toBe("");
+        expect(build.stdout.join("").trim()).toContain("K8s resource for the module saved");
+
+        // load helm manifests
+        const chartDir = `${testModule}/dist/${moduleConfig.uuid}-chart`;
+        const helm = await pepr.cli(chartDir, { cmd: `helm template .` });
+        expect(build.exitcode).toBe(0);
+        expect(build.stderr.join("").trim()).toBe("");
+        expect(build.stdout.join("").trim()).not.toBe("");
+
+        const helmManifest = `${chartDir}/reified.yaml`;
+        await fs.writeFile(helmManifest, helm.stdout.join("\n"));
+        helmResources = await resource.fromFile(helmManifest);
+
+        // load pepr manifests
+        const peprManifest = `${testModule}/dist/pepr-module-${moduleConfig.uuid}.yaml`;
+        peprResources = await resource.fromFile(peprManifest);
+      }, time.toMs("2m"));
+
+      it("peprVersion", async () => {
+        const peprStringified = JSON.stringify(peprResources);
+        const helmStringified = JSON.stringify(helmResources);
+
+        expect(peprStringified.includes(moduleConfig.peprVersion)).toBe(false);
+        expect(helmStringified.includes(moduleConfig.peprVersion)).toBe(false);
+      });
+
+      it("appVersion", async () => {
+        const peprStringified = JSON.stringify(peprResources);
+        const helmStringified = JSON.stringify(helmResources);
+
+        expect(peprStringified.includes(moduleConfig.appVersion)).toBe(false);
+        expect(helmStringified.includes(moduleConfig.appVersion)).toBe(false);
+      });
+
+      // eslint-disable-next-line max-statements, complexity
```

Comment:
```suggestion
      // eslint-disable-next-line max-statements, complexity
```

Refactoring this function adds more PR review overhead than it's worth at-present. I'd like to remove this linter suppression since `max-statements` and `complexity` are `warn` and not `error`.

The team has scheduled some time to talk about testing strategies next week. I'd prefer to not ignore the linter warnings and accept some warning output when working in this file (which we don't foresee changing very much anyways).